### PR TITLE
Update Github App documentation

### PR DESCRIPTION
**Pull request by ludydoo**
Content Read & Write is necessary for tide to check the merge status request
https://kubernetes.slack.com/archives/CDECRSC5U/p1635260608036300
https://kubernetes.slack.com/archives/CDECRSC5U/p1636581064092600

### CODE REVIEW

**Review by petr-muller — APPROVED** [review]
/ok-to-test

File: prow/getting_started_deploy.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Repository permissions:
 * Actions: Read-Only (Only needed when using the merge automation `tide`)
 * Administration: Read-Only (Required to fetch teams and collaborateurs)
 * Checks: Read-Only (Only needed when using the merge automation `tide`)
-* Contents: Read-Only
+* Contents: Read (Read & write needed when using the merge automation `tide`)
```
Comment (alvaroaleman) [inline prow/getting_started_deploy.md:22]: This is only the case when using GitHub enterprise or not?

**Comment by ludydoo** [comment]
No, I’m using a GitHub app for an organization but not gh enterprise

Le mar. 23 nov. 2021 à 15:16, Alvaro Aleman ***@***.***> a
écrit :

**Comment by alvaroaleman** [comment]
Oh wow, ok.
/lgtm
/approve

**Comment by ludydoo** [comment]
I know.. I filed a ticket with GitHub, it doesn’t seem right

Le mar. 23 nov. 2021 à 15:20, Alvaro Aleman ***@***.***> a
écrit :

**Comment by ludydoo** [comment]
Response from GitHub

Thanks for reaching out to GitHub Support!

The behaviour you have described is intentional for GitHub Apps -- This is gated internally and read permissions in our code is not enough to get these settings.

Write permissions will be needed on Contents to access the merge settings.

Hope this information helps!

Regards,

Oluwaseun
GitHub Support


Envoyé de mon iPhone

**Pull request merged**